wordpress-mobile/WordPress-Android
# Site Creation: Site Preview UI

Fixes #8505. This PR adds the UI for the site creation preview screen. It uses a mocked url which will be tied to the VM once we introduce it. If I haven't missed anything, it should be the exact replica of what @SylvesterWilmott has designed. Here is a screenshot from the emulator:

![screenshot_1542928702](https://user-images.githubusercontent.com/662023/48924172-6bb5c080-ee83-11e8-9ccc-3f32ae197d8e.png)

To test:
* The `NEW_SITE_CREATION_ENABLED` flag needs to be enabled in `WordPress/build.gradle`.
* Since the fragment is not added to the flow yet, it'll need to be done to be able to test. One easy way to do that would be to change the `showFragment` in `NewSiteCreationActivity.onCreate()` to instead show the newly added fragment. Once that's done, you can go to the site selection screen, tap on the `+` icon and select "Create WordPress.com site" to see it.
